### ISSUE TITLE
Error: uninitialized constant `PgLdapSync::Application::PGconn`

### ISSUE THREAD

**Issue by joshlk** [larskanis/pg-ldap-sync#18]
Hey,

From a fresh Ubuntu install, I get this error:
```
/var/lib/gems/2.3.0/gems/pg-ldap-sync-0.1.1/lib/pg_ldap_sync/application.rb:317:in `start!': uninitialized constant PgLdapSync::Application::PGconn (NameError)
	from /var/lib/gems/2.3.0/gems/pg-ldap-sync-0.1.1/lib/pg_ldap_sync/application.rb:353:in `run'
	from /var/lib/gems/2.3.0/gems/pg-ldap-sync-0.1.1/bin/pg_ldap_sync:6:in `<top (required)>'
	from /usr/local/bin/pg_ldap_sync:23:in `load'
	from /usr/local/bin/pg_ldap_sync:23:in `<main>'
```

My Ubuntu 16.04.03 setup was:
```
sudo apt-get install -y postgresql-server-dev-9.6 libpq-dev ruby ruby-all-dev
sudo gem install pg pg-ldap-sync
```

I then created a config file and ran:
```
pg_ldap_sync -c conf.yaml -vv
```

And I get the output (i've removed some confidential info):
```
I, [2018-01-19T16:38:00.504887 #5496]  INFO -- : found user-dn: ...
I, [2018-01-19T16:38:00.505617 #5496]  INFO -- : found user-dn: ...
I, [2018-01-19T16:38:00.508775 #5496]  INFO -- : found group-dn: ...
/var/lib/gems/2.3.0/gems/pg-ldap-sync-0.1.1/lib/pg_ldap_sync/application.rb:317:in `start!': uninitialized constant PgLdapSync::Application::PGconn (NameError)
	from /var/lib/gems/2.3.0/gems/pg-ldap-sync-0.1.1/lib/pg_ldap_sync/application.rb:353:in `run'
	from /var/lib/gems/2.3.0/gems/pg-ldap-sync-0.1.1/bin/pg_ldap_sync:6:in `<top (required)>'
	from /usr/local/bin/pg_ldap_sync:23:in `load'
	from /usr/local/bin/pg_ldap_sync:23:in `<main>'
```

Any help would be much appreciated! I've never used Ruby before so it's probably a noobie mistake.

**Comment by larskanis** [comment]
No, it's due to the fact that pg-1.0.0 removed some compatibility code, which pg-ldap-sync depends on. I'll fix this. For the time being you can do:
```sh
sudo gem uninst pg
sudo gem inst pg -v "~>0.14"
```
This installs the most resent version of the 0.x branch. Using pg-1.0.0 doesn't have any advantages currently.

**Comment by larskanis** [comment]
This is solved in pg-ladp-sync-0.2.0.
